src/dbt/kipptaf/models/csgf/rpt_csgf__enrollment.sql

```diff
@@ -0,0 +1,327 @@
+with
+    calendar_days as (
+        select yearid, schoolid, avg(days_total) as total_instructional_days,
+        from {{ ref("int_powerschool__calendar_rollup") }}
+        where yearid = {{ var("current_academic_year") - 1990 }}
+        group by yearid, schoolid
+    ),
+
+    students as (
+        select
+            schoolid,
+            student_number,
+            cast(grade_level as string) as grade_level_str,
+
+            case
+                race_ethnicity
+                when 'A'
+                then 'Asian'
+                when 'B'
+                then 'BL/AA'
+                when 'H'
+                then 'Hispanic or Latino'
+                when 'I'
+                then 'AI/AN'
+                when 'M'
+                then 'DTS'
+                when 'N'
+                then 'DTS'
+                when 'P'
+                then 'NH/OPI'
+                when 'T'
+                then '2+ races'
+                when 'W'
+                then 'White'
+                when 'Y'
+                then 'DTS'
+                else 'DTS'
+            end race_ethnicity,
+
+            if(lunch_status in ('F', 'f', 'R', 'FDC'), 1, 0) as frl,
+
+            if(grade_level = 0, 'K', cast(grade_level as string)) as grade_level,
+
+            if(iep_status = 'No IEP', 0, 1) as iep,
+
+            if(lep_status, 1, 0) as ell,
+
+        from {{ ref("int_extracts__student_enrollments") }}
+        where academic_year = {{ var("current_academic_year") }} and is_enrolled_oct01
```

Comment:
I think this needs an `rn_year = 1`. `is_enrolled_oct01` is windowed across every enrollment for a school year